lib/Doctrine/Inflector/Rules/French/Uninflected.php

```diff
@@ -29,6 +29,94 @@ public static function getPlural(): iterable
      */
     private static function getDefault(): iterable
     {
-        yield new Pattern('');
+        yield new Pattern('accès');
+        yield new Pattern('abus');
+        yield new Pattern('albatros');
+        yield new Pattern('anchois');
+        yield new Pattern('anglais');
```

Comment:
I guess this would make the list a lot longer but why not "danois", "hollandais", etc. etc.? @Alexandre-T hi! Maybe you would know since you contributed the original list to Symfony?